package.json

```diff
@@ -14,7 +14,7 @@
   },
   "main": "index.js",
   "peerDependencies": {
-    "eslint": "^3.3.0",
-    "eslint-plugin-react": "^6.0.0"
+    "eslint": "^4.14.0",
```

Comment:
note that this usually requires a small configuration change on the consumer end.

Reply:
Depending on the specific consumer, if they have extended our eslint rules, or used our old configuration, they'll have to follow the instructions here: https://stackoverflow.com/questions/44531243/eslint-module-build-failed-error-with-eslint-config-airbnb

Happened to me in a few repos. It has already been fixed here, but there may be outdated consumers.